dask/dataframe/core.py

```diff
@@ -1743,7 +1750,7 @@ def _describe_nonnumeric_1d(self, data, split_every=False):
             # count
             data.count(split_every=split_every),
             # most common value
-            vcounts.head(1, compute=False)
+            vcounts._head(1, npartitions=1, compute=False, safe=False)
```

Comment:
We use the private version to pass `safe=False`.

Now that I think, perhaps `unsafe=True/False` or `warn=True/False` is clearer than `safe`